### DISCUSSION TITLE
Building a collapsible Extended Floating Action Button with Jetpack Compose

### DISCUSSION THREAD

**Discussion by yujinyan** [yujinyan/blog#168] [Comments]

**Comment by wsdydeni** [comment]
你好，很喜欢这个添加了很多东西的主题，我如果直接clone仓库，需要改动哪些地方啊，谢谢

**Comment by yujinyan** [comment]
> 你好，很喜欢这个添加了很多东西的主题，我如果直接clone仓库，需要改动哪些地方啊，谢谢

你好，博客是用 Gatsby.js 开发的，了解这个框架的话应该会容易一些~

要修改的地方大概有
- gatsby-config.js 里的 `siteMetadata`、`gatsby-plugin-google-analytics` 等
- `bio.jsx` 
- src/pages 和 src/templates 和 src/components 里边有蛮多硬编码的字符串要替换
